src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -434,4 +430,13 @@ Valid from:</comment>
     <value>An attribute with "signing-certificate-v2" OID ({0}) is required.</value>
     <comment>0 - the expected OID value</comment>
   </data>
+  <data name="ErrorTimestampVerificationFailed" xml:space="preserve">
+    <value>Timestamp signature validation failed.</value>
+  </data>
+  <data name="TimestampCertificateFailsPublicKeyLengthRequirement" xml:space="preserve">
+    <value>The timestamp's certificate has a weak public key.</value>
```

Comment:
The timestamp certificate does not meet a minimum public key length requirement.